pages/docs/widgets/variants/calculator.en.mdx

```diff
@@ -1,3 +1,5 @@
+import { CategoryTable } from '@/components/categories/CategoryTable'
```

Comment:
With MDX, the file should start with YAML Frontmatter, not with a JS import. Otherwise the title will be parsed incorrectly and you'll end up with this (see the branch deployment):

<img width="329" alt="image" src="https://user-images.githubusercontent.com/596975/206447174-dcefcd91-0776-40ad-a3c4-dbad3be8904c.png">